datetime/src/main/java/com/fasterxml/jackson/datatype/jsr310/ser/key/ZonedDateTimeKeySerializer.java

```diff
@@ -25,7 +26,11 @@ public void serialize(ZonedDateTime value, JsonGenerator gen, SerializerProvider
          *
          * Keeping timezone info is a new feature which needs to be implemented.
```

Comment:
So now that it is implemented, the comment can be removed or changed, perhaps adding an @since?

Reply:
And maybe a question for @cowtowncoder but should this go in 2.10.1 or 2.11, and then master?

Reply:
Changed the comment, not sure where I should add the `@since`... or just remove it entirely?

Reply:
Comment looks good to me.

Reply:
Ok to remove / not add `@since` in this case.

As I said, 2.10 is ok: so, either base off 2.10 and I can merge forward. Or change in `master` and I can cherry-pick back. Slightly easier first way (start with 2.10) but not a big deal.